packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -1298,8 +1298,57 @@ void main() {
 
     // Padding is added to expanded header
     box = tester.renderObject(find.ancestor(of: find.byKey(firstPanelKey), matching: find.byType(AnimatedContainer)).first);
-    expect(box.size.height, equals(80.0)); // _kPanelHeaderCollapsedHeight + 32.0 (double default padding)
-    expect(box.size.width, equals(736.0));
+    expect(box.size.height, equals(80.0)); // _kPanelHeaderCollapsedHeight + 24.0 (double default padding)
+    expect(box.size.width, equals(744.0));
+  });
+
+  // Regression test for #5848
+  testWidgets('The AnimatedContainer and IconButton have the same height of 48px', (WidgetTester tester) async {
+    const Key firstPanelKey = Key('firstPanelKey');
+
+    await tester.pumpWidget(
+      const MaterialApp(
+        home: SingleChildScrollView(
+          child: SimpleExpansionPanelListTestWidget(
+            firstPanelKey: firstPanelKey,
+            canTapOnHeader: true,
+          ),
+        ),
+      ),
+    );
+
+    // The panel is closed
```

Comment:
Nit: Period at the end here.

Reply:
And in all of the other comments below.

Reply:
I copied the comment style from the other test cases to keep it consistent, but you are right - settings periods at the end of comments is part of the style guide! Thank you for the review.